bindings/client/src/app_websocket.rs

```diff
@@ -82,11 +80,24 @@ impl AppWebsocketInstrumented {
     }
 
     #[wind_tunnel_instrument(prefix = "app_")]
-    pub async fn network_info(
+    pub async fn dump_network_stats(&self) -> ConductorApiResult<kitsune2_api::TransportStats> {
```

Comment:
Prefer import over specifying type path unless there is a conflict
```suggestion
    pub async fn dump_network_stats(&self) -> ConductorApiResult<TransportStats> {
```

Reply:
[fixed](8c0f7014e3d498c4ffa4c7d1a40b6ff9995d853d)